archive/archivist/src/main/scala/uk/ac/wellcome/platform/archive/archivist/models/storage/ObjectMetadata.scala

```diff
@@ -0,0 +1,3 @@
+package uk.ac.wellcome.platform.archive.archivist.models.storage
+
+case class ObjectMetadata(userMetaData: Map[String, String])
```

Comment:
nit: let's be consistent with capitalisation.
```suggestion
case class ObjectMetadata(userMetadata: Map[String, String])
```